stetho/src/main/java/com/facebook/stetho/inspector/MethodDispatcher.java

```diff
@@ -63,6 +64,11 @@ public JSONObject dispatch(JsonRpcPeer peer, String methodName, @Nullable JSONOb
           "Not implemented: " + methodName,
           null /* data */));
     }
+
+    Stetho.getAnalyticsLogger().track(
+        "inspector_method_called",
+        Collections.singletonMap("name", methodName));
```

Comment:
This is not a HashMap. It is a SingletonMap. Looking at the source of java.util.Collections in Android shows what I would like to think is a near optimal implementation